aries_cloudagent/indy/models/proof.py

```diff
@@ -674,3 +674,11 @@ class IndyPresSpecSchema(AdminAPIMessageTracingSchema):
         required=False,
         example=False,
     )
+    auto_remove = fields.Bool(
+        description=(
+            "Whether to remove the presentation exchange record on completion "
+            "(overrides --preserve-exchange-records configuration setting)"
+        ),
+        required=False,
+        default=False,
+    )
```

Comment:
This feels misplaced. Why is this on an indy specific model?

Reply:
maybe there is a bad/inappropriate inheritance structure...

but that schema is the request schema for `/present-proof/records/{pres_ex_id}/send-presentation`...

yeah, some ugly stuff going on there.

Reply:
great catch though.  i don't want to get into a complete re-engineering here but will create a new schema using the IndyPresSpecSchema as a parent and add the `auto_remove` to the child.  Should be a step in the right direction and put the `auto_remove` where it is actually used.